hoard/sources/whoas.py

```diff
@@ -44,25 +45,16 @@ def __next__(self) -> Dataset:
 
 
 def create_from_whoas_dim_xml(data: str) -> Dataset:
-    kwargs: Dict[str, Any] = {}
+    kwargs: DefaultDict[str, Any] = defaultdict(list)
```

Comment:
What was the reasoning behind switching to a defaultdict here? I ask because there are several fields in this dictionary whose values are supposed to be strings.

Reply:
Helen's recommendation here https://mitlibraries.atlassian.net/browse/RDRP-365

Reply:
I think we should instead use `setdefault` for fields that are lists and just keep this as a plain ol' dictionary.